c7n/resources/redshift.py

```diff
@@ -245,6 +245,38 @@ def process_cluster_snapshot(self, cluster):
             ClusterIdentifier=cluster['ClusterIdentifier'])
 
 
+@actions.register('enhanced-vpc-routing')
```

Comment:
sounds good albeit a bit wordy, what if we just dropped the enhanced, and called it `enable-vpc-routing`